components/clp-py-utils/clp_py_utils/core.py

```diff
@@ -62,6 +65,26 @@ def read_yaml_config_file(yaml_config_file_path: pathlib.Path):
     return config
 
 
+def resolve_host_path(path: pathlib.Path) -> pathlib.Path:
+    """
+    Translates a host path to its container-mount equivalent.
+
+    :param path: The host path.
+    :return: The translated container path.
+    """
+    path = path.absolute()
+    resolved = CONTAINER_HOST_ROOT_DIR / path.relative_to("/")
+
+    try:
+        if resolved.is_symlink():
```

Comment:
this is needed because somehow my WSL Ubuntu has this
```
(clp) junhao@ASUS-X870E:~$ ls -l .aws
lrwxrwxrwx 1 junhao junhao 24 Aug 26 12:07 .aws -> /mnt/c/Users/junhao/.aws
```

`/mnt/c/Users/junhao/.aws` is certainly not a valid path in the runtime container, to access, the bootstrap scripts should use the prefix + `/mnt/c/Users/junhao/.aws`